public/app/features/dashboard-scene/scene/UnconfiguredPanel.tsx

```diff
@@ -92,20 +104,30 @@ function UnconfiguredPanelComp(props: PanelProps) {
     );
   }
 
+  const { isEditing } = dashboard.state;
+
   return (
     <Stack direction={'row'} alignItems={'center'} height={'100%'} justifyContent={'center'}>
       <Box paddingBottom={2}>
-        <ButtonGroup>
-          <Button icon="sliders-v-alt" onClick={onConfigure}>
-            <Trans i18nKey="dashboard.new-panel.configure-button">Configure</Trans>
-          </Button>
-          <Dropdown overlay={MenuActions} placement="bottom-end" onVisibleChange={onMenuClick}>
-            <Button
-              aria-label={t('dashboard.new-panel.configure-button-menu', 'Toggle menu')}
-              icon={isOpen ? 'angle-up' : 'angle-down'}
-            />
-          </Dropdown>
-        </ButtonGroup>
+        {isEditing ? (
+          <ButtonGroup>
+            <Button icon="sliders-v-alt" onClick={onConfigure}>
+              <Trans i18nKey="dashboard.new-panel.configure-button">Configure</Trans>
+            </Button>
+            <Dropdown overlay={MenuActions} placement="bottom-end" onVisibleChange={onMenuClick}>
+              <Button
+                aria-label={t('dashboard.new-panel.configure-button-menu', 'Toggle menu')}
+                icon={isOpen ? 'angle-up' : 'angle-down'}
+              />
+            </Dropdown>
+          </ButtonGroup>
+        ) : (
+          <EmptyState
+            variant="call-to-action"
+            message={t('dashboard.new-panel.missing-config', 'Missing panel configuration')}
```

Comment:
Wasn't sure whether we aligned on ["Unconfigured"](https://github.com/grafana/grafana/issues/115233#issuecomment-3646746363) or "Missing panel configuration"
Other than that LGTM!